crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -288,14 +312,19 @@ impl DiskFileSystem {
         let delay = Duration::from_millis(1);
         // Create a watcher object, delivering debounced events.
         // The notification back-end is selected based on the platform.
-        let mut watcher = watcher(tx, delay)?;
+        let mut debounced_watcher = notify_debouncer_full::new_debouncer(delay, None, tx)?;
         // Add a path to be watched. All files and directories at that path and
         // below will be monitored for changes.
         #[cfg(any(target_os = "macos", target_os = "windows"))]
-        watcher.watch(&root_path, RecursiveMode::Recursive)?;
+        debounced_watcher
+            .watcher()
+            .watch(&root_path, RecursiveMode::Recursive)?;
```

Comment:
I think we also want the "cache" here and below?

`debouncer.cache().add_root(&root_path, RecursiveMode::Recursive);`

Reply:
full debouncer internally autosetup those

https://github.com/notify-rs/notify/blob/08e74dae8e96fbd25704cdaa530ffc02f6d33039/CHANGELOG.md#L9-L20

Reply:
but the version in this PR is too old